verification/cbmc_stubs/semaphore.c

```
@@ -2,7 +2,7 @@
 
 #include <semaphore.h>
 
-int sem_init(sem_t *sem, int pshared, unsigned int value)
+int _avocado_sem_init(sem_t *sem, int pshared, unsigned int value)
```

Clang (14.0.6)

[warning] declaration uses identifier '_avocado_sem_init', which is reserved in the global namespace

(bugprone-reserved-identifier)